From: Kevin Kofler <kevin.kofler@chello.at>
Date: Sat, 17 Sep 2011 16:18:24 +0000
Subject: K3b::IsOverburnAllowed: Increase tolerance from 10% to 25%.
X-Git-Url: http://quickgit.kde.org/?p=k3b.git&a=commitdiff&h=b0df9e7771ac9148004187bb53ba13ac40b01cae
---
K3b::IsOverburnAllowed: Increase tolerance from 10% to 25%.

10% are not enough to allow using CD-R90 and CD-R99 media to full capacity.
The new 25% tolerance allows using those media while still catching the common
case of DVD ISO vs. CD media.

BUG: 276002
REVIEW: 102648
---

From: Kevin Kofler <kevin.kofler@chello.at>
Date: Sat, 17 Sep 2011 21:34:23 +0000
Subject: K3b::IsOverburnAllowed: Don't count used capacity twice.
X-Git-Url: http://quickgit.kde.org/?p=k3b.git&a=commitdiff&h=8270d2853f2962facdf6f9ea6d669922d6c78aba
---
K3b::IsOverburnAllowed: Don't count used capacity twice.

CCBUG: 276002
---


--- a/libk3b/core/k3bglobals.cpp
+++ b/libk3b/core/k3bglobals.cpp
@@ -595,7 +595,7 @@
 bool K3b::IsOverburnAllowed( const Msf& projectSize, const Msf& capacity, const Msf& usedCapacity )
 {
     return( k3bcore->globalSettings()->overburn() &&
-        (projectSize + usedCapacity) <= ( capacity.lba() - usedCapacity.lba() + capacity.lba() * 10 / 100 ) ); // 10% tolerance in overburn mode
+        (projectSize + usedCapacity) <= ( capacity.lba() + capacity.lba() / 4 ) ); // 25% tolerance in overburn mode
 }
 
 

